web/src/lib/components/layouts/page-header.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+  export let title: string | undefined = undefined;
+  export let description: string | undefined = undefined;
+
+  export let scrollbar = true;
+
+  $: scrollbarClass = scrollbar ? '' : 'h-full';
+</script>
+
+{#if title}
+  <div
+    class="sticky top-0 z-[100] bg-immich-bg dark:bg-immich-dark-bg flex w-full place-items-center justify-between border-b p-4 dark:border-immich-dark-gray dark:text-immich-dark-fg"
+  >
+    <div class="flex gap-2 items-center">
+      <div class="font-medium">{title}</div>
+      {#if description}
+        <p class="text-sm text-gray-400 dark:text-gray-600">{description}</p>
+      {/if}
+    </div>
+    <slot name="buttons" />
+  </div>
+{/if}
+<div class={scrollbarClass}>
+  <slot />
+</div>
```

Comment:
This should not be in PageHeader component imo.

Reply:
Oh, you just want the title and the buttons slot in the page-header right ?